models/calendar.yml

```diff
@@ -81,7 +81,19 @@
         title: Introductions
         emoji: 👋
 
-    - 7:45 PM: Speaker TBA
+    - 7:45 PM:
+     type: Talk
+        title: Host Your Own Katy Perry Party
+        emoji: :information_desk_person:
```

Comment:
Could you replace this with the literal emoji? As in 💁  instead of `:information_desk_person:`